src/+otp/+kpr/KPRProblem.m

```diff
@@ -24,5 +24,13 @@ function onSettingsChanged(obj)
             obj.RHSSlow = otp.RHS(@(t,y) otp.kpr.fSlow(t, y, lambda, omega), ...
                 'Jacobian', @(t, y) otp.kpr.jacobianSlow(t, y, lambda, omega));
         end
+        
+        function y = internalSolveExactly(obj, t)
+            if ~isequal(obj.Y0, [2; sqrt(3)])
```

Comment:
sqrt(3) might not be exact, so approx equal? Is there a standard matlab function for equal to some precision?

Reply:
This one will be tricky for VPA and gpuArray, but I think that's out of scope for now. Honestly, I think it's good enough for now. Surprisingly it handles cases like `isequal(sqrt(3), single(sqrt(3)))`